### PR TITLE
[HSEARCH] 6.2.0.Final

### PR DESCRIPTION
**Pull request by yrodiere**
Most content for the "What's new" section copied from [this page on the website](https://hibernate.org/search/releases/6.2/#whats-new).

Staging: https://staging.in.relation.to/2023/07/07/hibernate-search-6-2-0-Final/

### CODE REVIEW

File: posts/Yoann/2023-07-07-hibernate-search-6-2-0-Final.adoc [hunk 1]
```diff
@@ -0,0 +1,522 @@
...
+
+Its key feature compared to the Hibernate ORM integration is its ability to run without Hibernate ORM or a relational database.
+It can be used to index entities coming from an arbitrary datastore or even
+(though that’s not recommended in general) to use Lucene or Elasticsearch as a primary datastore.
+
+For more information about the Standalone POJO Mapper,
+see link:{hsearch-doc-url-prefix}#mapper-pojo-standalone[this section of the reference documentation].
+
+To get started with the Standalone POJO Mapper,
+see link:{hsearch-doc-url-prefix}#mapper-pojo-standalone-getting-started[this getting started guide].
```
Comment (marko-bekhta) [inline posts/Yoann/2023-07-07-hibernate-search-6-2-0-Final.adoc]: maybe let's use the new link to the guide here too ?

File: posts/Yoann/2023-07-07-hibernate-search-6-2-0-Final.adoc [hunk 2]
```diff
@@ -0,0 +1,522 @@
...
+for more information.
+
+[[indexing-plan-filter]]
+=== Indexing plan filters
+
+With link:{hsearch-jira-url-prefix}/HSEARCH-168[HSEARCH-168]/link:{hsearch-jira-url-prefix}/HSEARCH-1383[HSEARCH-1383],
+Hibernate Search now allow enabling/disabling indexing in indexing plans (which include in particular automatic indexing),
+both fully (for all types) or on a per-type basis.
+
+For example, the following will disable automatic indexing
```
Comment (marko-bekhta) [inline posts/Yoann/2023-07-07-hibernate-search-6-2-0-Final.adoc:184]: I am now triggered by `automatic indexing` 😆

**Review by marko-bekhta — APPROVED** [review]
looks nice 🎉

File: posts/Yoann/2023-07-07-hibernate-search-6-2-0-Final.adoc (hunk 2 above)
Comment (yrodiere) [reply posts/Yoann/2023-07-07-hibernate-search-6-2-0-Final.adoc:184]: > I am now triggered

I see what you did here

**Comment by yrodiere** [comment]
I addressed your comments, thanks!

File: posts/Yoann/2023-07-07-hibernate-search-6-2-0-Final.adoc [hunk 3]
```diff
@@ -0,0 +1,522 @@
...
+See link:{hsearch-doc-url-prefix}#indexing-plan-filter[this section of the reference documentation]
+for more information.
+
+[[mapping-improvements]]
+=== Mapping improvements
+
+`@IndexedEmbedded(excludePaths = ...)`::
+The `@IndexedEmbedded` annotation now exposes an `excludePaths` attribute,
+allowing the inclusion of all paths with only a few selectively excluded,
+as opposed to the previous approach of selectively including paths with `includePaths`.
```
Comment (gsmet) [inline posts/Yoann/2023-07-07-hibernate-search-6-2-0-Final.adoc]: Is it clear that the `previous approach` is still available?

**Review by gsmet — COMMENTED** [review]
I added a very small comment.

File: posts/Yoann/2023-07-07-hibernate-search-6-2-0-Final.adoc (hunk 3 above)
Comment (yrodiere) [reply posts/Yoann/2023-07-07-hibernate-search-6-2-0-Final.adoc]: Changed to `pre-existing`, thanks.

**Pull request merged**